exporter/info.go

```diff
@@ -249,13 +249,24 @@ func (e *Exporter) handleMetricsReplication(ch chan<- prometheus.Metric, masterH
 }
 
 func (e *Exporter) handleMetricsRocksDB(ch chan<- prometheus.Metric, fieldKey string, fieldValue string) {
+	sharedMetric := []string{"block_cache_usage"}
+	for _, field := range sharedMetric {
+		// format like `block_cache_usage:0`
+		if strings.Compare(fieldKey, field) == 0 {
+			if statValue, err := strconv.ParseFloat(fieldValue, 64); err == nil {
+				e.registerConstMetricGauge(ch, fieldKey, statValue)
+			}
+			// return ASAP
+			return
+		}
+	}
+
 	prefixs := []string{
-		"block_cache_usage", "block_cache_pinned_usage", "index_and_filter_cache_usage", "estimate_keys",
-		"level0_file_limit_slowdown", "level0_file_limit_stop", "pending_compaction_bytes_slowdown",
+		"block_cache_pinned_usage", "index_and_filter_cache_usage", "estimate_keys", "level0_file_limit_slowdown", "level0_file_limit_stop", "pending_compaction_bytes_slowdown",
```

Comment:
@jjz921024 Guess we can keep `block_cache_usage` here to be compatible with the old version of Kvrocks server.

Reply:
Yes, update it, please review